src/ngx_http_lua_ssl_certby.c

```diff
@@ -201,6 +201,10 @@ ngx_http_lua_ssl_cert_handler(ngx_ssl_conn_t *ssl_conn, void *data)
 
     c = ngx_ssl_get_connection(ssl_conn);
 
+    if (c->ssl->renegotiation) {
+        return 0;
```

Comment:
IMHO, we should `return -1` and log with error message.
In the ideal world, in which Nginx and OpenSSL work perfectly, server side renegotiation is disable.
Also, skip the `ssl_cert_by_lua` phase and use the dummy ssl certificate is not always expected in the renegotiation.

Reply:
Why should we return `-1`? `-1` represents the current stage is still in progress.

Reply:
@tokers 
My bad. Yes, it should return `0`.

Reply:
@spacewander 
I have updated this change.